packages/bpk-component-calendar/src/BpkCalendarGrid.js

```diff
@@ -189,6 +192,7 @@ BpkCalendarGrid.defaultProps = {
   showWeekendSeparator: true,
   weekStartsOn: 1,
   ignoreOutsideDate: false,
+  dateProps: {},
```

Comment:
The default should just be null :).
```suggestion
  dateProps: null,
```